en/docs/develop-components/ballerina-private-package-usage.md

```diff
@@ -0,0 +1,34 @@
+# Ballerina Private Package Usage
+
+Ballerina private packages allow developers to share the reusable code within the organization via Ballerina central without making it available to the public.
+Choreo enables you to use Ballerina private packages when creating components using the Ballerina build preset.
+
+In order to use Ballerina private packages, you need to make sure that you are using the same organization in Choreo and Ballerina Central.
+For example, if you are using `choreo` as the organization name in Choreo, following conditions should be met. 
+
+- The `Ballerina.toml` in the private package should have the `choreo` as the organization name.
+    ```
+    [package]
+    org = "choreo"
+    name = "greeting_lib"
+    ```
+- The package should be published to the Ballerina Central using a token that is generated for the `choreo` organization.
+
+    !!! tip
+        You can refer to the [Publish packages to Ballerina Central](https://ballerina.io/learn/publish-packages-to-ballerina-central/) documentation to learn how to publish a Ballerina package to the Ballerina Central.
+    
+
+- The `Ballerina.toml` in the application should have the `choreo` as the organization name.
```

Comment:
Shall we have this as sequential steps. And it will be great if we can create a flow where a user would want to use this in a scenario in Choreo.

Reply:
should have the `choreo` -> should have `choreo`